contrib/epee/include/storages/levin_abstract_invoke2.h

```diff
@@ -34,10 +34,26 @@
 #undef MONERO_DEFAULT_LOG_CATEGORY
 #define MONERO_DEFAULT_LOG_CATEGORY "net"
 
+namespace
+{
+  template<typename context_t>
+  void on_levin_traffic(const context_t &context, bool initiator, bool sent, bool error, size_t bytes, const char *category)
+  {
+    MCINFO("net.p2p.traffic", context << bytes << " bytes " << (sent ? "sent" : "received") << (error ? "/corrupt" : "")
```

Comment:
I meant to comment on this too - this level seems too low. We're writing to the log constantly just for bandwidth tracking stuff. If its that important, having the information tracked internally and then dumped periodically (either to log or elsewhere) should be a lot faster.

Reply:
It is not on by default.